dotnet/src/Functions/Functions.UnitTests/Connectors/WebApi/Rest/RestApiOperationTests.cs

```diff
@@ -87,13 +87,14 @@ public void ShouldReplacePathParametersByValuesFromArguments()
     }
 
     [Fact]
-    public void ShouldReplacePathParametersByDefaultValues()
+    public void ItShouldReplacePathParametersByDefaultValues()
     {
         // Arrange
         var parameterMetadata = new RestApiOperationParameter(
             "fake-path-parameter",
             "fake_type",
             true,
+            false,
```

Comment:
nit: name arguments for readability and maintainability

Reply:
Fixed.